elastic/kibana
# [SecuritySolution] Show upselling component for entity risk score tab

## Summary

### Before
Serverless: The tab is hidden

![Screenshot 2024-05-16 at 15 57 36](https://github.com/elastic/kibana/assets/1490444/bd270106-70a7-452e-9b22-5a3960a615a3)

ESS: We displayed the unauthorized banner

![Screenshot 2024-05-16 at 15 38 24](https://github.com/elastic/kibana/assets/1490444/164a16ad-08d4-49b6-b592-ee9ae91cd196)

### After
Display the upsell banner
![Screenshot 2024-05-16 at 14 46 11](https://github.com/elastic/kibana/assets/1490444/a64593aa-789c-4db4-8747-68b5d5e51435)


## How to test it

* For every license and tier 
  * Go to host/user page inside, explore menu item
  * Verify what is displayed inside the risk score tab


* Run security serverless with "Security Analytics Essentials"  tier
  * You should see the upsell component
* Run security serverless with "Security Analytics Complete"  tier
  * You should NOT see the upsell component
* Run kibana ESS with platinum license
  * You should NOT see the upsell component
* Run kibana ESS with basic license
  * You should see the upsell component


### Checklist

Delete any items that are not applicable to this PR.
